tests/system/action/projector/test_create.py

```diff
@@ -0,0 +1,65 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorCreateActionTest(BaseActionTestCase):
+    def test_create_correct(self) -> None:
+        self.set_models(
+            {
+                "meeting/222": {"name": "name_SNLGsvIV"},
+            }
+        )
+        response = self.request(
+            "projector.create",
+            {
+                "name": "test projector",
+                "meeting_id": 222,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "projector/1", {"name": "test projector", "meeting_id": 222}
+        )
```

Comment:
Can you check, that the defaults are correct?

Reply:
Done